Dockerfile

```
@@ -65,40 +65,67 @@ RUN ls -la rel/overlays/
+# Ensure static files are properly copied in the release
+RUN mkdir -p /app/lib/wanderer_notifier-${APP_VERSION}/priv/static/app && \
+    cp -r priv/static/app/* /app/lib/wanderer_notifier-${APP_VERSION}/priv/static/app/ && \
+    echo "Verifying files in release directory:" && \
+    ls -la /app/lib/wanderer_notifier-${APP_VERSION}/priv/static/app/
+
 # Compile and build release
 RUN mix compile --warnings-as-errors && \
     mix release --overwrite
 
+# Debug: Show release version (optional)
+RUN find _build/prod/rel/wanderer_notifier/lib -name "wanderer_notifier-*.ez" | head -n1 | sed 's/.*wanderer_notifier-\(.*\)\.ez/\1/' > /tmp/app_version.txt && \
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)